webapp-django/crashstats/manage/jinja2/manage/status_message.html

```diff
@@ -0,0 +1,62 @@
+{% extends "manage/base.html" %}
+
+{% block page_title %}{{ super() }} - Status Message{% endblock %}
+
+{% block site_js %}
+{{ super() }}
+{% javascript 'moment' %}
+{% javascript 'pagination' %}
+{% javascript 'api_tokens' %}
+{% endblock %}
+
+{% block site_css %}
+{{ super() }}
+{% stylesheet 'api_tokens' %}
+{% endblock %}
+
+{% block admin_title %}{{ super() }} - Status Message{% endblock %}
+
+{% block mainbody %}
+<div class="panel">
+    <div class="body notitle">
+
+        <h3>Current Active Status Messages</h3>
+
+        <table class="data-table vertical">
+            <thead>
+                <tr>
+                    <th>Message</th>
+                    <th>Severity</th>
+                    <th>&nbsp;</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for status in statuses %}
+                <tr>
+                    <td>{{ status.message }}</td>
+                    <td>{{ status.severity }}</td>
+                    <td><a href="{{ url('manage:status_message_disable') }}?id={{ status.id }}">Disable</a></td>
```

Comment:
That's not what you did for your other admin apps. And I'm getting more and more reluctant to putting arguments in the URL. It makes sense for some things, but not all, and I find there is a beauty in query strings.